packages/node-bridge/helpers.ts

```diff
@@ -295,13 +295,15 @@ export function createServerWithHelpers(
       res.send = body => send(req, res, body);
       res.json = jsonBody => json(req, res, jsonBody);
 
-      await handler(req, res);
-    } catch (err) {
-      if (err instanceof ApiError) {
-        sendError(res, err.statusCode, err.message);
-      } else {
-        throw err;
+      try {
+        await handler(req, res);
+      } catch (err) {
+        console.log(`Rejected Promise returned from ${req.url}: ${err.stack}`);
```

Comment:
```suggestion
        console.log(`Error from API Route ${req.url}: ${err.stack}`);
```

Nit pick: the user's code isn't always a promise because it isnt always async

Reply:
Done! [`e5c05b6` (#9410)](https://github.com/vercel/vercel/pull/9410/commits/e5c05b61ddffc51ae71f94db4ec0c98de86c2e49)